src/pages/claims/claims.tsx

```diff
@@ -0,0 +1,65 @@
+import { ReactNode } from 'react';
+import { connectWallet } from '../../components/sign-in/sign-in';
+import Layout from '../../components/layout';
+import Button from '../../components/button';
+import BorderedBox, { Header } from '../../components/bordered-box';
+import ClaimList from './claim-list';
+import { useChainData } from '../../chain-data';
+import { useActiveClaims } from '../../logic/claims';
+import styles from './claims.module.scss';
+
+export default function Claims() {
```

Comment:
Nit: I prefer using arrow function everywhere (for consistency). Not that it would matter that much though.

Reply:
haha we had this discussion with the API3 team a while back. I (personally) still like the old school `function` keyword for 2 reasons: 

1. makes it easier to spot functions at a glance (imo)
2. hoisting

But the outcome of the team discussion was that most people prefer arrow functions so we went with that going forward

Reply:
Haha yeah I also much prefer function statements. In addition to your 2 points
1. it allows "same line" default exports (which is less typing and makes it immediately obvious when you read a component whether it is the default export for the file):

```tsx
export default function Component(props: Props) {
```
vs
```tsx
const Component = (props: Props) => {
  // many lines of code
}

export default Component;
```
2. Less typing:

```ts
function fn(arg: string) {
```
vs
```ts
const fn = (arg: string) => {
```
3. When you need generic types in your component props, then you are faced with syntax errors with arrow functions as it clashes with JSX:

```tsx
function Component<T>(props: Props<T>) { // All good
```
```tsx
const Component = <T>(props: Props<T>) => { // Syntax error: JSX element 'T' has no corresponding closing tag
```
You can use an awkward comma to fix the syntax error though:
```tsx
const Component = <T,>(props: Props<T>) => { // Awkward, but works
```

Reply:
Makes sense. Feel free to use whatever you prefer.